18F/identity-idp
# Break account reactivation into two-step process

Previously, account reactivation occurred on a single screen. We are now moving it to two screens to better match the  identity verification flow. 

New personal key entry screen:
![screen shot 2017-06-08 at 11 39 52 am](https://user-images.githubusercontent.com/1421848/26937389-3ca009aa-4c3f-11e7-999d-7fb66b211a48.png)

Manual verification:
0). Make sure you have a personal key for the account in question
1). Reset the password of an LOA3 account
2). Log back in
3). Click 'I have my personal key' on the reactivate account screen
4). Enter your personal key
5). On the next screen, enter your password
6). You should be redirected to the account page, with a new personal key, as before

I'm also planning to refactor the user_session logic to use a service class, similar to the idv session implementation. If people think it makes more sense to include in this PR, I can go back and add it in!
